sources/main.cpp

```diff
@@ -98,9 +98,12 @@ main(int argc, char* argv[]) {
     }
 
     context.exportOutputs();
-
     context.execute();
 
+    dfl::common::Log::init(options, config.outputDir(), true);
+
+    LOG(info) << MESS(SimulationEnded, context.basename()) << LOG_ENDL;
```

Comment:
This means that we don't give any information in the console when simulation is ended ?

Reply:
Depends on Dynawo implementation but not in DFL

Reply:
There is another ticket about ease of use and logs of DFL

Reply:
ok because we add the experience with dynawo never saying Simulation finished OK or NOK in the console before and it is better to have a small message for the user at the end of the simulation in the console. if it is addressed later on then no problem.

Reply:
yes, the MR is almost ready :)